apps/examples/ble_perfs/ble_perfs_main.c

```diff
@@ -0,0 +1,452 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <tinyara/timer.h>
+#include <wifi_manager/wifi_manager.h>
+#include <ble_manager/ble_manager.h>
+#include <errno.h>
+
+#define BLE_PERF_TAG "[BLE_PERFS] "
+#define BLE_LOGD(fmt, args...) printf(BLE_PERF_TAG " %s:(%d) " fmt, __FUNCTION__, __LINE__, ##args)
+#define BLE_LOGE(fmt, args...) printf(BLE_PERF_TAG " [ERR] %s:(%d) " fmt, __FUNCTION__, __LINE__, ##args)
+
+#define BLE_APP_HANDLE_SERVICE_0 (0x006b)
+#define BLE_STATE_MANAGER_RMC_HANDLE_KEY_COMMAND (0x006c)
+#define BLE_STATE_MANAGER_RMC_HANDLE_KEY_CCCD (0x006e)
+
+#define BLE_APP_HANDLE_SERVICE_1 (0x0073)
+#define BLE_APP_HANDLE_CHAR_RMC_KEY (0x0074)
+#define BLE_APP_HANDLE_DESC_RMC_KEY (0x0076)
+#define BLE_APP_HANDLE_SERVICE_2 (0x0077)
+#define BLE_APP_HANDLE_CHAR_RMC_SYNC (0x0078)
+#define BLE_MAX_MTU 240
+#define MAX_PACKET_COUNT 100
+#define MAX_PACKET_SIZE BLE_MAX_MTU - 3; // 237 bytes
+#define FIRST_PACKET 11
+#define NEXT_PACKET 55
+#define LAST_PACKET 99
```

Comment:
Could you alignt these macro definitions in the way that it's been done for some other files in this repository?

Reply:
I did not get. Please provide the reference

Reply:
I think he meant 

```
#define abcd       1
#define pqr        2
#define lmnopq     3
```